client/components/deposits-overview/recent-deposits-list.tsx

```diff
@@ -66,6 +69,39 @@ const RecentDepositsList: React.FC< RecentDepositsProps > = ( {
 	if ( isLoading || recentDeposits.deposits.length === 0 ) {
 		return <></>;
 	}
+
+	// Add a notice indicating the potential business day delay for pending and in_transit deposits.
+	let bannerAdded = false;
+	const depositRows = recentDeposits.deposits
+		.reverse() // reverse the array so that the oldest pending or in_transit deposit has the notice added to it.
```

Comment:
I don't see any immediate side-effect here, but since `reverse()` mutates the array which is defined outside of this component's scope, I suggest duplicating the array before mutating ([react docs](https://react.dev/learn/updating-arrays-in-state#updating-arrays-without-mutation)).
```suggestion
	const depositRows = [ ...recentDeposits.deposits ]
		.reverse() // reverse the array so that the oldest pending or in_transit deposit has the notice added to it.
```

Reply:
Oh thanks. I was actually going to request specific feedback about this. I wasn't sure if there was a better way to acheive this.

Reply:
This _might_ be able to be improved by finding the ID of the last deposit _before_ executing the "render" loop, for example:

```tsx
	const lastPendingOrInTransitDepositId = [ ...recentDeposits.deposits ]
		.reverse()
		.find(
			( deposit ) =>
				'pending' === deposit.status || 'in_transit' === deposit.status
		)?.id;

	const depositRows = recentDeposits.deposits.map( ( deposit ) => (
		<Fragment key={ deposit.id }>
			<DepositTableRow deposit={ deposit } />
			{ deposit.id === lastPendingOrInTransitDepositId && (
				<BannerNotice
					status="info"
					icon={ <InfoOutlineIcon /> }
					children={ strings.notices.businessDayDelay }
					isDismissible={ false }
				/>
			) }
		</Fragment>
	) );
```